File: 2st/Yeonwoo/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    pPcount = 0
    yYcount = 0
    for py in s:
        if py=='p' or py == 'P':
            pPcount += 1
        elif py=='y' or py == 'Y':
            yYcount += 1
    if pPcount == yYcount:
        answer = True
    elif pPcount != yYcount:
        answer = False
            
    
    # [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    #print('Hello Python')

    return answer

File: 3st/Hyeonseo/programmer/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int p = 0;
        int y = 0;
        char[] ch = s.toCharArray();
        for(int i = 0; i < ch.length; i++) {
            if(ch[i] == 'p' || ch[i] == 'P') {
                p++;
            }
            else if(ch[i] == 'y' || ch[i] == 'Y') {
                y++;
            }
        }
        if(p != y) answer = false; 
        return answer;
    }
}
